### ISSUE TITLE
Failed to receive frame from AudioSocket message forchannel

### ISSUE THREAD

**Issue by irishabh96** [matak/audiosocket#1]
Failed to receive frame from AudioSocket message forchannel 
Getting this error. How to resolve this?

**Comment by jayasingh1802** [comment]
Same here. Getting below response:
- <SIP/voda-00000004> Playing 'beep.alaw' (language 'en')
    -- Executing [s@STREAM:5] AudioSocket("SIP/voda-00000004", "40325ec2-5efd-4bd3-805f-53576e581d13,127.0.0.1:4000") in new stack
[Feb 16 15:47:21] WARNING[80712][C-00000005]: res_audiosocket.c:264 ast_audiosocket_receive_frame: Received non-audio AudioSocket message
[Feb 16 15:47:21] ERROR[80712][C-00000005]: res_audiosocket.c:302 ast_audiosocket_receive_frame: Insufficient data read from AudioSocket
[Feb 16 15:47:21] ERROR[80712][C-00000005]: app_audiosocket.c:206 audiosocket_run: Failed to receive frame from AudioSocket message forchannel SIP/voda-00000004

**Comment by Moondev0607** [comment]
I don't think this is a working example, as, to begin with, AudioSocket is actually expecting a socket not a web server.
